make-android

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
I think in other repos this is called `build-android`

Reply:
So better to keep the standard? `build-android`?